Tasks/ServiceFabricUpdateAppVersions/Update-PackageVersion.psm1

```diff
@@ -77,7 +77,7 @@
                     # Search the package files for differences (Unless the user doesn't want to log all changes and we already know the package xml changed)
                     $newPackagePath = Join-Path $NewPackageRoot $packageName
                     $oldPackagePath = Join-Path $OldPackageRoot $packageName
-                    if (Find-FileChanges $newPackagePath $oldPackagePath $LogIndent -LogAllChanges:$LogAllChanges)
+                    if ((Test-Path $newPackagePath) -and (Find-FileChanges $newPackagePath $oldPackagePath $LogIndent -LogAllChanges:$LogAllChanges))
```

Comment:
What about a scenario(in increasing order of time):
1. Deploy application with a service having config package version 1.0
2. Upgrade application with same service having no config package ==> this would be fine as there is  no new package and hence version does not matter
3, Deploy application with same service having new config package ==> no old package exists, but we would still need to bump up version to 2.0(or whatever)

I think this scenario is very corner and it is okay if we don't want to solve it.